docs/site/DEVELOPING.md

```diff
@@ -286,6 +286,48 @@ LoopBack 4 documentation is hosted inside this monorepo in the
 This allows us to change both implementation and the documentation in a single
 pull request.
 
+### Organization of content
+
+LoopBack 4 is a large codebase organized into dozens of packages. We,
+developers, are often tempted to include a lot of details and wider context when
+writing docs. While this is good for capturing the knowledge in a text form, it
+also makes the documentation difficult to follow by people new to LoopBack.
+Newcomers get quickly overwhelmed by the amount of concepts they need to
+understand and all the different packages they need to know about.
+
+To prevent that issue, we have categorized all monorepo packages into two
+groups:
+
+1. Building-blocks are lower-level packages that are typically not consumed by
+   LoopBack projects directly, instead there is a higher-level package exposing
```

Comment:
`LoopBack projects` -> `LoopBack applications` or `LoopBack application projects`.

Reply:
Well, I think this applies to most extension projects too, but let's not dwell on details. I am going to use "LoopBack applications" for now, we can improve this later as we get a better picture about the documentation for extension authors.